### PR TITLE
new sigin page

### PR DESCRIPTION
**Pull request by vinayhnindium**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -14,3 +14,25 @@ code {
...
+    @font-face {
+      font-family: "lato-heavy";
+      src: local("lato-heavy"),
+       url("./assest/fonts/Lato-Heavy.ttf") format("truetype");
+      }
+      @font-face {
+        font-family: "lato-Medium";
+        src: local("lato-Medium"),
+         url("./assest/fonts/Lato-Medium.ttf") format("truetype");
+        }
```
Comment (maheshnanubala) [inline src/index.css:38]: is the indentation correct here ?

File: src/pages/Signin/Signin.js [hunk 2]
```diff
@@ -1,67 +1,119 @@
...
+const { register, handleSubmit } = useForm();
 
-  const onSubmit = (formValues, e) => {
+const onSubmit = (formValues, e) => {
     e.preventDefault();
-    if (formValues.email && formValues.password) {
-      dispatch(signin({ formValues, navigate, toast }));
+      if (formValues.email && formValues.password) {
+    dispatch(signin({ formValues, navigate, toast }));
     }
```
Comment (maheshnanubala) [inline src/pages/Signin/Signin.js:23]: Indentation

File: src/pages/Signin/Signin.js [hunk 3]
```diff
@@ -1,67 +1,119 @@
...
+          <Container>
+            <Row id='heading-text' className="justify-content-center align-items-center mt-5" >
+              BOOKIT
+            </Row> 
+          </Container>
+          <Card className="form-cotainer p-4 mt-5 ">
+            <Form onSubmit={handleSubmit(onSubmit)}> 
+              <Row>
+              <p id='signin-text'> Sign-in</p>
+              </Row>
```
Comment (maheshnanubala) [inline src/pages/Signin/Signin.js:41]: Indentation

File: src/pages/Signin/Signin.js [hunk 4]
```diff
@@ -1,67 +1,119 @@
...
+// <button type="submit" className="btn btn-primary signInButton">
+// Sign-in
+// </button>
+// </div>
+// <p className="signUpLink">
+// <a href="#">Sign-up</a>
+// </p>
+// </form>
+// </div>
+);
```
Comment (maheshnanubala) [inline src/pages/Signin/Signin.js:111]: remove if not required

File: src/pages/Signin/Signin.scss [hunk 5]
```diff
@@ -1,39 +1,167 @@
...
+      height: 45px;
+  } 
+  #heading-text{
+      margin-bottom: 18vh;
+      font-size: 32px;
+  }
+  #navlinks{
+      padding: 5px 50px 20px 50px;
   }
+
```
Comment (maheshnanubala) [inline src/pages/Signin/Signin.scss:166]: is this empty line required ?

**Review by maheshnanubala — CHANGES_REQUESTED** [review]

**Comment by vinayhnindium** [comment]
to make changes

**Pull request closed without merging**